docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+services:
+  app:
```

Comment:
This probably doesn't super-matter but I wonder if it'd make sense to call this service `server` instead of `app`?